src/containers/Review/ReviewPage.tsx

```diff
@@ -66,19 +66,26 @@ const ReviewPage: React.FC<{
 
   const { addScrollable, scrollTo } = useScrollableAttachments()
 
-  const [showWarningModal, setShowWarningModal] = useState<ModalProps>({ open: false })
+  const { ConfirmModal: WarningModal, showModal: showWarningModal } = useConfirmationModal({
+    type: 'warning',
+    title: strings.REVIEW_STATUS_PENDING_TITLE,
+    message: strings.REVIEW_STATUS_PENDING_MESSAGE,
+  })
+
+  useEffect(() => {
+    if (!reviewStructure) return
+    if (thisReview?.current.reviewStatus === ReviewStatus.Pending) {
+      console.log('Should show modal')
+      showWarningModal({
+        onConfirm: () => push(`/application/${reviewStructure.info.serial}/review`),
```

Comment:
this seems like a one button modal

Reply:
Yeah good point. I just added an additional property to hide the cancel button and applied it here (and a couple of others) 

13317a0e